website/public/install/index.ejs.html

```diff
@@ -137,7 +137,12 @@
       }
 
       .long-shell-command-line {
+<<<<<<< HEAD
         /* 6 = "$ ".length + 4 spaces of indentation */
+=======
+        /* 4 = "$ ".length + 4 spaces of indentation */
+        display: inline-block;
+>>>>>>> fixed validation errors for /install page
```

Comment:
Blocking: Don't commit merge markers.

Reply:
Just removed them, thank you for you patience!